target/linux/ath79/image/nand.mk

```diff
@@ -231,6 +231,23 @@ define Device/netgear_ath79_nand
   UBINIZE_OPTS := -E 5
 endef
 
+define Device/netgear_pgzng1
+  SOC := ar9344
+  DEVICE_MODEL := PGZNG1
+  DEVICE_VENDOR := NETGEAR
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ledtrig-usbport kmod-i2c-gpio \
+  kmod-leds-pca955x kmod-rtc-isl1208 kmod-spi-dev uboot-envtools
```

Comment:
one extra indent please (or is it there and this github view hides it?). 

uboot-envtools is part of the DEFAULT_PACKAGES of the ath79 target.